src/preprocessor.py

```diff
@@ -918,4 +1056,4 @@ class DefaultPreprocessor(DaskMultiFilePreprocessor):
     """Implementation class for :class:`MDTFPreprocessorBase` for the general
     use case. Includes all implemented functionality and handles multi-file data.
     """
-    _file_preproc_functions = []
+    file_preproc_functions = []
```

Comment:
Move variable name mods to separate PR

Reply:
Resolved in [130e994](https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/372/commits/130e994fcaf3a78a0ec4188e2a375e4c0f5cc5d9), see https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/372#discussion_r897516962